battle-service/src/main/java/org/astashonok/battleservice/rest/BattleController.java

```diff
@@ -0,0 +1,44 @@
+package org.astashonok.battleservice.rest;
+
+import org.astashonok.battleservice.entities.Battle;
+import org.astashonok.battleservice.models.BattleInfo;
+import org.astashonok.battleservice.models.BattleCreationForm;
+import org.astashonok.battleservice.models.MoveForm;
+import org.astashonok.battleservice.services.BattleService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/battles")
+public class BattleController {
+
+    @Autowired
+    private BattleService battleService;
+
+    @PostMapping("/{battleId}/make-move")
+    public void makeMove(@RequestBody MoveForm moveForm) {
+        battleService.makeMove(moveForm);
+    }
+
+    @GetMapping("/opened")
```

Comment:
you are returning DB objects(entities) which is bad practice, due to not all DB fields potentially required for the client.  you should think about DTO contracts